sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ContactService.java

```diff
@@ -151,9 +147,30 @@ public Exposure buildGeneratedExposureDataForContact() {
         .contactToBodyFluids(YesNoUnknownOptions.NO)
         .handlingSamples(YesNoUnknownOptions.NO)
         .typeOfPlace(TypeOfPlace.HOME)
-        .continent("Africa")
-        .subcontinent("Central Africa")
-        .country("Cameroon")
+        .typeOfPlaceDetails("Some type of place details")
```

Comment:
use faker to generate random data, we need to validate in UI if this field is changed in the future tests